client/hotrod-client/src/main/java/org/infinispan/client/hotrod/multimap/RemoteMultimapCache.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.client.hotrod.multimap;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.multimap.api.BasicMultimapCache;
+
+/**
+ * {@inheritDoc}
+ * <p>
+ * Remote MultimapCache interface used for server mode.
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @see <a href="http://infinispan.org/documentation/">Infinispan documentation</a>
+ * @since 9.2
+ */
+public interface RemoteMultimapCache<K, V> extends BasicMultimapCache<K, V> {
+
+   CompletableFuture<MetadataCollection<V>> getWithMetadata(K key);
```

Comment:
I am thinking if we indeed want to return the metadata, maybe we should have a way to modify it. Do we want a `replaceWithVersion` or some way to override the default expiration parameters?

Reply:
Also please document the method.

Reply:
@wburns maybe, I'm going to log a ticket on that

Reply:
https://issues.jboss.org/browse/ISPN-8582